cluster/kubevirtci.sh

```diff
@@ -12,8 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.18'}
-export KUBEVIRTCI_VERSION=${KUBEVIRTCI_VERSION:-'463da29b9cbdcaa22daed3a8ef5c5e864f582b0b'}
+export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.20'}
```

Comment:
I see you have an old KubeVirtCI version in your `_kubevirtci` folder checked in. I figure for this to work you first need to remove your `_kubevirtci` folder once, then it should update it  to the latest version for you.

```
Done setting up docker in docker.                                                                                                                                                                                                                                                                     
+ /bin/sh -c 'export TARGET=k8s-1.20 && automation/test.sh'                                                                                                                                                                                                                                           
+ export PATH=/bin:/google-cloud-sdk/bin:/workspace:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin                                                                                                                                                                    
+ PATH=/bin:/google-cloud-sdk/bin:/workspace:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin                                                                                                                                                                           
+ export KUBEVIRT_PROVIDER=k8s-1.20                                                                                                                                                                                                                                                                   
+ KUBEVIRT_PROVIDER=k8s-1.20                                                                                                                                                                                                                                                                          
+ export KUBEVIRT_WITH_CNAO=true                                                                                                                                                                                                                                                                      
+ KUBEVIRT_WITH_CNAO=true                                                                                                                                                                                                                                                                             
+ export IMAGEIO_NAMESPACE=cdi                                                                                                                                                                                                                                                                        
+ IMAGEIO_NAMESPACE=cdi                                                                                                                                                                                                                                                                               
+ export VCSIM_NAMESPACE=cdi                                                                                                                                                                                                                                                                          
+ VCSIM_NAMESPACE=cdi                                                                                                                                                                                                                                                                                 
+ export KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1                                                                                                                                                                                                                                            
+ KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1                                                                                                                                                                                                                                                   
+ make cluster-down                                                                                                                                                                                                                                                                                   
./cluster/down.sh                                                                                                                                                                                                                                                                                     
+ source ./cluster/kubevirtci.sh                                         
++ export KUBEVIRT_PROVIDER=k8s-1.20
++ KUBEVIRT_PROVIDER=k8s-1.20                                            
++ export KUBEVIRTCI_VERSION=2108081530-91f55e3
++ KUBEVIRTCI_VERSION=2108081530-91f55e3
++ export KUBEVIRTCI_TAG=2108081530-91f55e3
++ KUBEVIRTCI_TAG=2108081530-91f55e3
++ KUBEVIRTCI_PATH=/home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci
+ kubevirtci::install                                                    
+ '[' '!' -d /home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci ']'
++ kubevirtci::path                                                      
++ echo -n /home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci
+ /home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci/cluster-up/down.sh
/home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci/cluster-up/down.sh: line 11: /home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci/cluster-up//cluster/k8s-1.20/provider.sh: No such file or directory
/home/prow/go/src/github.com/kubevirt/vm-import-operator/_kubevirtci/cluster-up/down.sh: line 12: down: command not found
Makefile:135: recipe for target 'cluster-down' failed
make: *** [cluster-down] Error 127                                       
+ EXIT_VALUE=2                                                           
+ set +o xtrace                                                          
Cleaning up after docker in docker.                                      
================================================================================
[Barnacle] 2021/09/10 09:18:46 Cleaning up Docker data root...
```

Reply:
This repo doesn't have a checked in `_kubevirtci` folder, and when I run the same test locally it succeeds. I'm not sure why it would have failed for you.